### PR TITLE
Back compatibility with tus js version

### PR DESCRIPTION
**Pull request by pkochubey**
Go client not worked with tus js server implementation

### CODE REVIEW

**Comment by skrater** [comment]
Thanks @pkochubey i will take a look.

File: client.go [hunk 1]
```diff
@@ -97,6 +98,20 @@ func (c *Client) CreateUpload(u *Upload) (*Uploader, error) {
...
+		baseUrl, err:= netUrl.Parse(c.Url)
+		if err != nil{
+			return nil, ErrUrlNotRecognized
+		}
+
+		newUrl, err:= netUrl.Parse(url)
+		if err != nil{
+			return nil, ErrUrlNotRecognized
+		}
+		if newUrl.Scheme == ""{
```
Comment (skrater) [inline client.go]: add space before `{`

File: client.go [hunk 2]
```diff
@@ -97,6 +98,20 @@ func (c *Client) CreateUpload(u *Upload) (*Uploader, error) {
 	case 201:
 		url := res.Header.Get("Location")
 
+		baseUrl, err:= netUrl.Parse(c.Url)
+		if err != nil{
+			return nil, ErrUrlNotRecognized
+		}
+
+		newUrl, err:= netUrl.Parse(url)
+		if err != nil{
```
Comment (skrater) [inline client.go]: add space before `{`

File: client.go [hunk 3]
```diff
@@ -97,6 +98,20 @@ func (c *Client) CreateUpload(u *Upload) (*Uploader, error) {
 	case 201:
 		url := res.Header.Get("Location")
 
+		baseUrl, err:= netUrl.Parse(c.Url)
+		if err != nil{
```
Comment (skrater) [inline client.go]: add space before `{`

**Comment by jaswdr** [comment]
LGTM :+1:  @pkochubey thanks for the PR, only need to run `goimports` or `go fmt`

**Comment by pkochubey** [comment]
Added new commit after run go fmt

**Pull request merged**